### ISSUE TITLE
Endless decompilation after replacing two parameters with "Set Equate..."

### ISSUE THREAD

**Issue by Damok82** [NationalSecurityAgency/ghidra#4158]
**Describe the bug**
Replacing two parameters with "Set Equate..." on the same function triggers an endless decompilation.

**To Reproduce**
1. Analyse and decompile a program.
 ```CPP
ExampleFunction((char *)0x0,(char *)0x0);
```
2. Click on one of the function's parameters and select 'Set Equate...".
3. Select NULL and change the parameter.
4. Click on the other parameter of the function and select 'Set Equate..." again.
5. Select NULL and change the parameter.

**Example program**
```CPP
// Compile with 'g++ GhidraEquateExample.cpp -O0 -o GhidraEquateExample'

#include <iostream>
void ExampleFunction(char*, char*);
int main()
{
    ExampleFunction(NULL, NULL);
}
void ExampleFunction(char*, char*)
{
}
```

**Expected behavior**
Both parameters are changed to NULL.

**Environment (please complete the following information):**
 - OS: Microsoft Windows 10 Pro 64-bit 21H2 19044.1645
 - Java Version: 17.0.1
 - Ghidra Version: Ghidra 10.1.2 Build PUBLIC 2022-Jan-25 1526 EST
 - Ghidra Origin: official ghidra-sre.org distro

**Additional context**
Changing only one parameter works fine.
Changing and resetting one parameter and then changing the other parameter works fine, too.
Only after changing both parameter the endless decompilation is triggered.
Changing both parameters to two different values does also not trigger the endless decompilation.

**Comment by ryanmkurtz** [comment]
This is a duplicate of #4121, which will be fixed in the upcoming patch release.
